packages/esm-patient-programs-app/src/programs/programs-form.test.tsx

```diff
@@ -80,20 +80,14 @@ describe('ProgramsForm', () => {
     renderProgramsForm();
 
     const programNameInput = screen.getByRole('combobox', { name: /program name/i });
-    // FIXME: Make the date input work
-    const enrollmentDateInput = screen.getByLabelText(/date enrolled/i);
-    // const enrollmentDateDayInput = within(enrollmentDateInput).getByRole('spinbutton', { name: /day/i });
-    // const enrollmentDateMonthInput = within(enrollmentDateInput).getByRole('spinbutton', { name: /month/i });
-    // const enrollmentDateYearInput = within(enrollmentDateInput).getByRole('spinbutton', { name: /year/i });
+    const enrollmentDateInput = screen.getByRole('textbox', { name: /date enrolled/i });
     const enrollmentLocationInput = screen.getByRole('combobox', { name: /enrollment location/i });
     const enrollButton = screen.getByRole('button', { name: /save and close/i });
 
     await user.click(enrollButton);
     expect(screen.getByText(/program is required/i)).toBeInTheDocument();
 
-    // await user.type(enrollmentDateDayInput, '05');
-    // await user.type(enrollmentDateMonthInput, '05');
-    // await user.type(enrollmentDateYearInput, '2020');
+    fireEvent.change(enrollmentDateInput, { target: { value: '2020-05-05' } });
```

Comment:
For some reason, `user.paste` as I did in the conditions test, doesn't work here.